pkg/system/db_reconciler.go

```diff
@@ -163,11 +149,25 @@ func (r *Reconciler) reconcileDBCluster() error {
 
 	// Apply changes to the cluster resources. create or update the modified cluster
 	if r.CNPGCluster.UID == "" {
-		// Cluster resource was not created yet. Create it and handle import if needed
 
-		if err := r.reconcileClusterCreateOrImport(); err != nil {
-			r.cnpgLogError("got error setting up cluster import. error: %v", err)
-			return err
+		// Cluster resource is missing. Check if noobaa CR has a recovery configuration.
+		if r.NooBaa.Spec.DBSpec.DBRecovery == nil {
+			// No recovery configuration found, set bootstrap configuration to init a new DB
+			if r.CNPGCluster.Spec.Bootstrap == nil {
+				r.CNPGCluster.Spec.Bootstrap = &cnpgv1.BootstrapConfiguration{
+					InitDB: &cnpgv1.BootstrapInitDB{
+						Database:      noobaaDBName,
+						Owner:         noobaaDBUser,
+						LocaleCollate: "C",
+					},
+				}
+			}
+		} else {
+			// Recovery configuration found, set bootstrap configuration to recover from the snapshot
```

Comment:
This is because user explicitly deleted the cluster, right?
If so I would add it to the comment.